applications/blockpublisher.md

```diff
@@ -0,0 +1,235 @@
+Has experience in substrate, rust, reactjs, leptos python and also in building mobile apps in react native and tauri.
+
+Ipsita Pradhan (Ph.D), is a GIS analyst and developer, and has two year experince in rust, substrate, polkadot, and leptos.
+
+Tanushree Behera (B.Ed) has one and half year experinece in rust, substrate, polkadot and leptos.
+
+### Team Code Repos
+
+- https://github.com/orgs/reaudito/repositories
+
+### Overview
+- **Total Estimated Duration:** 1 Years and 8 months
+- **Full-Time Equivalent (FTE):**  2 FTE
+- **Total Costs:** 30,000 USD
+- **DOT %:** 50% Dot
+
+## Milestone:
+
+
+### Milestone 1:
+
+- Substrate Template Development:
+
+Develop Substrate templates for price discovery, article storage, and staking coin management.
+
+
+- **Estimated duration:** 8 month
+- **FTE:**  1,3
+- **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  GPL |
+| 0b. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user about the project |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article**/workshop that explains our project (what was done/achieved as part of the grant.) |
+| 1. | Basic schema | Develop Substrate templates for price discovery, article storage, and staking coin management.|
+| 2. | Tests | Implement and test the Substrate templates to ensure seamless integration with the overall system.|
+
+### Milestone 2:
+
+- Leptos UI Development:
+
+Design and develop a user-friendly UI for article upload, viewing, and interaction with the Substrate templates using Leptos.
+Integrate the Leptos UI with the Substrate templates to enable a seamless user experience.
+
+- **Estimated duration:** 6 month
+- **FTE:**  1,3
+- **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  GPL |
+| 0b. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user about the project |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article**/workshop that explains our project (what was done/achieved as part of the grant.) |
+| 1. | Leptos UI Development | Design and develop a user-friendly UI for article upload, viewing, and interaction with the Substrate templates using Leptos.|
```

Comment:
Do you foresee any challenges in building a Rust front-end instead of React or some other TS framework?

Reply:
Subxt works seamlessly with the Rust WASM (Leptos) framework for interacting with Substrate chains. Furthermore, one can use JavaScript packages through WASM Bindgen if necessary. Rust crates and packages are growing rapidly to meet almost all needs. Therefore, I don't foresee any challenges in using a Rust-based frontend. It is also easier to train newcomers, as they only need to learn and work with a single language—Rust. The compile-time checks in Rust contribute to a more stable frontend when using the Rust framework.